azurerm/internal/services/frontdoor/tests/frontdoor_firewall_policy_resource_test.go

```diff
@@ -336,6 +337,32 @@ resource "azurerm_frontdoor_firewall_policy" "test" {
     }
   }
 
+  custom_rule {
+    name                           = "Rule3"
+    enabled                        = true
+    priority                       = 2
```

Comment:
looks like the priority needs to be different here:

```
------- Stdout: -------
=== RUN   TestAccAzureRMFrontDoorFirewallPolicy_complete
=== PAUSE TestAccAzureRMFrontDoorFirewallPolicy_complete
=== CONT  TestAccAzureRMFrontDoorFirewallPolicy_complete
TestAccAzureRMFrontDoorFirewallPolicy_complete: testing.go:684: Step 0 error: errors during apply:
Error: creating Front Door Firewall Policy "testAccFrontDoorWAF200826081908186558" (Resource Group "testaccRG-200826081908186558"): frontdoor.PoliciesClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="BadRequest" Message="WebApplicationFirewallPolicy validation failed. More information \"Custom rules Rule2 and Rule3 have duplicate priority: 2\"."
```

could we update this so that the tests can pass?

Reply:
This should now be resolved also, and I think I've found the right values for the environment variables required to run acceptance tests locally in future.